ftplugin/yaml/vim_vault_inline.vim

```diff
@@ -15,15 +15,15 @@
 " --------------------------------
 " Add our plugin to the path
 " --------------------------------
-python import sys
-python import vim
-python sys.path.append(vim.eval('expand("<sfile>:h")'))
+py3 import sys
```

Comment:
I think you can this as multi-version compatible by using `if has("python")` syntax:

```
if has("python")
    # Do the previous imports
elseif has("python3")
    # Do the new imports
else
    # Print an error about not having python compiled in
endif
```